librabbitmq/amqp_openssl_bio.c

```diff
@@ -147,7 +147,7 @@ int amqp_openssl_bio_init(void) {
     return AMQP_STATUS_NO_MEMORY;
   }
 
-  BIO_METHOD *meth = BIO_s_socket();
+  BIO_METHOD *meth = (BIO_METHOD *)BIO_s_socket();
```

Comment:
In my variant, I added a comment that the necessity of this cast is a known issue of openssl.

Reply:
I add too.